src/openapi3/handleJson.test.ts

```diff
@@ -1352,23 +1438,26 @@ const parsed = [
         range: null,
         reference: null,
         required: false,
+        embedded: null,
         description: "Author the author of the review",
       },
     ],
   },
 ];
 
 describe(`Parse OpenApi v3 Documentation from Json`, () => {
-  const toBeParsed = handleJson(
-    openApi3Definition,
-    "https://demo.api-platform.com"
-  );
+  test(`Properties to be equal`, async () => {
+    const toBeParsed = await handleJson(
+      openApi3Definition,
+      "https://demo.api-platform.com"
+    );
 
-  test(`Properties to be equal`, () => {
     expect(toBeParsed[0].name).toBe(parsed[0].name);
     expect(toBeParsed[0].url).toBe(parsed[0].url);
     expect(toBeParsed[0].id).toBe(parsed[0].id);
 
+    expect((toBeParsed[0].fields as Field[])[0]).toEqual(parsed[0].fields[0]);
```

Comment:
It shouldn't be needed to cast `fields`.

Reply:
Ah, OK - actually just copied the other version. :)

Reply:
Actually, if I remove that cast I get this error:
`src/openapi3/handleJson.test.ts:1465:12 - error TS2533: Object is possibly 'null' or 'undefined'.`

Reply:
You need to write it like this:
```suggestion
    expect(toBeParsed[1].fields?.[0]).toEqual(parsed[1].fields[0]);
```